library-backend/library-api/src/main/java/org/internship/library/api/BookService.java

```diff
@@ -6,31 +6,39 @@
 public interface BookService {
     /**
      * Method without implementation for retrieving a book
```

Comment:
It's obivous this methods don't have implementation. This is not the purpose of the comments.
The purpose of the javadoc is to briefly explain what that method will do.